Quantum-Control-Applications/Superconducting/Single Flux Tunable Transmon/SSB mixer for readout/README.md

```diff
@@ -0,0 +1,25 @@
+# Single flux tunable transmon with SSB mixer for readout
+
+<img align="right" src="SIngle Flux Tunable Transmon Setup.PNG" alt="drawing" width="400"/>
+
+## Experimental setup and context
+
+These files showcase various experiments that can be done on an single flux tunable transmon where the readout pulses 
+are sent through a SSB mixer. Standard IQ mixers are used for qubit adressing and signal downconversion before acquisition.
+
+These files were tested in a real setup shown on on the right, but are given as-is with no guarantee.
+
+While these can serve as a template for new labs or for new experiments, certain adaptations will probably have to be made.
+Use with care.
+
+## Basic Files
+1. [Hello_qua](hello_qua.py) - A script used for basic qua program demonstration.
+2. [Look_at_raw_traces](look_at_raw_traces.py) - A script for acquiring raw ADC traces from inputs 1 and 2 and check ADC saturation and time of flight.

```

Comment:
Not sure. I like time_of_flight, but we are doing more than just time of flight... Whatever you feel like, let's just make it consistent :)

* I will say, instead of "look at raw traces" I would write "raw_adc_traces"